README.adoc

```diff
@@ -208,6 +177,125 @@ When the service is ready, you will see an output similar to the one below. The
 
 The JSON output has a `highest` attribute which represents the system with the highest load. Similarly, the `lowest` attribute represents the system with the lowest load. The JSON output for each of these feature the respective hostname and systemLoad of the system.
 
+When you are done checking out the application, run the following command to stop the query microservice. We will leave the `system` and `inventory` services running because we will use them when we rebuild the application later on. 
+
+--
+[role='command']
+```
+docker stop query
+```
+--
+
+
+== Updating the web client to use an alternative reactive provider
+
+pom.xml
+[source, xml, linenums, role="code_column"]
+----
+include::finish/query/pom.xml[]
+----
+
+Although Open Liberty provides the Apache CXF JAX-RS reactive provider by default, you can alternatively use another provider, such as Eclipse Jersey or RESTEasy, which also support the use of https://github.com/ReactiveX/RxJava[RxJava^] reactive libraries. 
```

Comment:
Same as before.
```suggestion
Although Open Liberty provides the default reactive provider which returns `CompletionStage` types, you  can alternatively use another provider that supports the use of other reactive frameworks like https://github.com/ReactiveX/RxJava[RxJava^] .  The Apache CXF and Eclipse Jersey projects produce such providers.
```

While RESTEasy likely also produces reactive providers, we have not tested them with Liberty. It is possible that RESTEasy's providers will only work with a RESTEasy JAX-RS runtime (like Wildfly or Quarkus), so that is why I don't recommend that we mention RESTEasy here.  It is certainly possible that RESTEasy's provider would work with Liberty with no problems, but I wouldn't want to advertise it until we have tested it.